instructlab.ai.zone

```diff
@@ -42,7 +42,8 @@ google._domainkey                    IN TXT "v=DKIM1; k=rsa; p=MIIBIjANBgkqhkiG9
 google._domainkey                    IN TXT "nWDWqwcNuDNrSiowi1QYQO1V3pcfpCKZCpOo3L3jprRP9ojndblrGmoC5Kp9Ffu7AB12+TPqE71e36CqjpMTvSxZY8jsel6YKmCG62CxleSQBg9o0Qq9c9v8dJ7rS7JoW14wR9kT8XMjcTaT/jwJqLSIev+"
 google._domainkey                    IN TXT "j7ZQIEZKuZDSyGwogEV7m3SuFNUIMFVoHrdNNGhN+pFjB+P9Cnw30V01k51l7zm3C5HRsa3rQsfg/oc3xBEk9nzvIJVOf3wIDAQAB"
 
-qa.ui   IN              CNAME  qa-ui-instructlab-ai-0e3e0ef4c9c6d831e8aa6fe01f33bfc4-0000.us-south.containers.appdomain.cloud.
-ui      IN              CNAME  ui-instructlab-ai-0e3e0ef4c9c6d831e8aa6fe01f33bfc4-0000.us-south.containers.appdomain.cloud.
-docs	IN		CNAME  instructlab.github.io.
-blog    IN              CNAME  instructlab.wpenginepowered.com.
+qa.ui   	IN              CNAME  qa-ui-instructlab-ai-0e3e0ef4c9c6d831e8aa6fe01f33bfc4-0000.us-south.containers.appdomain.cloud.
+ui      	IN              CNAME  ui-instructlab-ai-0e3e0ef4c9c6d831e8aa6fe01f33bfc4-0000.us-south.containers.appdomain.cloud.
+docs		IN		CNAME  instructlab.github.io.
+blog    	IN              CNAME  instructlab.wpenginepowered.com.
+research	IN		CNAME  xukai92.github.io.
```

Comment:
```suggestion
research	IN		CNAME  Red-Hat-AI-Innovation-Team.github.io.
```

Reply:
I created an org for our team and moved our GH Page repo to it.
Hopefully the subdir is not needed any more too.